src/Sema.zig

```diff
@@ -7090,9 +7090,28 @@ fn zirEmbedFile(sema: *Sema, block: *Scope.Block, inst: Zir.Inst.Index) CompileE
 }
 
 fn zirErrorName(sema: *Sema, block: *Scope.Block, inst: Zir.Inst.Index) CompileError!Air.Inst.Ref {
+    const tracy = trace(@src());
+    defer tracy.end();
+
     const inst_data = sema.code.instructions.items(.data)[inst].un_node;
     const src = inst_data.src();
-    return sema.mod.fail(&block.base, src, "TODO: Sema.zirErrorName", .{});
+    _ = src;
```

Comment:
Out of curiosity what is the purpose of this assignment?

`src` is always used in the if below

Reply:
Thanks for the review, I am cleaning up someone else's code, will remove.